pageserver/src/http/openapi_spec.yml

```diff
@@ -1384,6 +1384,18 @@ components:
           type: string
           format: hex
 
+    LsnByTimestampResponse:
+      type: object
+      required:
+        - lsn
+        - kind
+      properties:
+        lsn:
+          type: string
+          format: hex
+        kind:
+          type: string
```

Comment:
NIT: I think kind should be a enum with description, what each value means

Reply:
made it an enum.